### DISCUSSION TITLE
How do I open the software after extracting from zip?

### DISCUSSION THREAD

**Discussion by jamesr2k** [Klocman/Bulk-Crap-Uninstaller#660] [General]
Sorry for being so new, but this is something I've always struggled with when it comes to github.  There's nothing that's clear about how to actually open the application itself.  I've looked through the BCU_manual and it does a fantastic job of saying what's in the software, but not how to actually open it once the files are extracted from the zip/rar.

Is it in the source folder? The es application doesn't seem to do anything.  Is it in the main folder? I don't see anything looks like an executable or a .bat in there.  Completely stuck and would really appreciate some help with this noob question, thanks for your time!

**Comment by hrxn** [comment]
![scrn01](https://github.com/user-attachments/assets/425ecd95-a3b1-40f0-a326-535e1595e47f)
![scrn02](https://github.com/user-attachments/assets/850ac26a-2e1c-42d0-a57c-8d6fb1f2c7ca)

**Reply by jamesr2k** [reply]
Thanks a ton!  Much appreciated
